test/include/test_runner/csv_converter.h

```diff
@@ -7,30 +7,30 @@
 namespace kuzu {
 namespace testing {
 
-// Convert an entire CSV dataset directory to parquet.
+// Convert an entire CSV dataset directory to some specified output.
```

Comment:
I would suggest moving the csv->json class to a separate header and source files.
And have a base common class CSVConverter.

Reply:
csv->parquet and csv->json aren't different classes. The only part of the code that actually defines the class to convert to `parquet` is a `".parquet"` file extension added on top of export destinations. The only real change here is that I changed the hardcoded `".parquet"` to an `outputFileExtension` variable.